CMakeLists.txt

```diff
@@ -455,6 +455,12 @@ if (HAVE_EXECINFO_H)
   set (HAVE_STACKTRACE 1)
 endif (HAVE_EXECINFO_H)
 
+if (WIN32)
+  set (HAVE_STACKTRACE 1)
+  set (HAVE_SYMBOLIZE 1)
+  target_link_libraries (glog PUBLIC Dbghelp.lib)
```

Comment:
I would remove the `.lib` suffix here and replace `Dbghelp` by proper capitalization of library's name, i.e., `DbgHelp `.